bgpd/bgp_evpn_vty.c

```diff
@@ -2999,6 +2999,130 @@ DEFUN (no_bgp_evpn_default_originate,
 	return CMD_SUCCESS;
 }
 
+DEFPY (dup_addr_detection,
+       dup_addr_detection_cmd,
+       "dup-addr-detection [max-moves (2-1000)$max_moves_val time (2-1800)$time_val]",
+       "Duplicate address detection\n"
+       "Max allowed moved before address detected as duplicate\n"
```

Comment:
Shouldn't this be "Max allowed moves before address detected as duplicate\n" ? i.e. moves instead of moved.

Reply:
yes, I will change to max allowed moves